.github/workflows/ingest-test-fixtures-update-pr.yml

```diff
@@ -43,36 +28,10 @@ jobs:
     needs: [setup]
     steps:
       - uses: actions/checkout@v3
-      - uses: actions/cache/restore@v3
-        id: base-virtualenv-cache
-        with:
-          path: |
-            .venv
-            nltk_data
-          key: unstructured-${{ runner.os }}-${{ env.PYTHON_VERSION }}-${{ env.GHA_CACHE_KEY_VERSION }}-${{ hashFiles('requirements/*.txt') }}
-      # Due to the dependency on setup, the cache should exist before this is ran. Set failed if it wasn't found.
-      - name: Setup base virtual environment (no cache hit)
-        if: steps.base-virtualenv-cache.outputs.cache-hit != 'true'
-        uses: actions/github-script@v3
+      - uses: ./.github/actions/base-ingest-cache
         with:
-          script: |
-            core.setFailed("base cached environment couldn't be found")
-      - uses: actions/cache@v3
-        id: virtualenv-cache
-        with:
-          path: |
-            .venv
-            nltk_data
-          key: unstructured-ingest-${{ runner.os }}-${{ env.PYTHON_VERSION }}-${{ env.GHA_CACHE_KEY_VERSION }}-${{ hashFiles('requirements/ingest/*.txt') }}
-      - name: Set up Python ${{ env.python-version }}
-        uses: actions/setup-python@v4
-        with:
-          python-version: ${{ env.python-version }}
-      - name: Setup virtual environment (no cache hit)
-        run: |
-          python${{ env.python-version }} -m venv .venv
-          source .venv/bin/activate
-          make install-all-ingest
+          python-version: ${{ env.PYTHON_VERSION }}
+          check-only: 'true'
```

Comment:
I'm confused..if we check-only and it DOES exist, it looks like we don't actually restore it. I'm assuming I'm not following some piece of this...